system/system_monitor/reader/hdd_reader/hdd_reader.cpp

```diff
@@ -47,28 +47,26 @@
 
 #include "hdd_reader/hdd_reader.hpp"
 
-namespace fs = boost::filesystem;
-
 // 7634-7647 Unassigned
 constexpr int PORT = 7635;
 
 /**
  * @brief HDD information
  */
-typedef struct
+using HDD_Info = struct
```

Comment:
or just `struct HDD_Info {};` ?

Reply:
Oh, yeah! Fixed in 9a9612f83ad86dc2f070485d189f567a317b0f97.
`packed` still works.
https://wandbox.org/permlink/hZxGP08FUcCF1BLP